std_msgs/Header header
gvgexplore/Ridge[] ridges
gvgexplore/Ridge close_ridge
gvgexplore/Pixel[] pixels